app/services/cdl/eligible_item_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Cdl
+  class EligibleItemService
```

Comment:
I think a class-level comment that says what this class is for would be nice here.